plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -751,8 +759,171 @@ bool MeshImportContext::applyEdgeCreases()
+  {
+
+    const UsdGeomPrimvar& primvar = *it;
+    TfToken name, interpolation;
+    SdfValueTypeName typeName;
+    int elementSize;
+
+    primvar.GetDeclarationInfo(&name, &typeName, &interpolation, &elementSize);
+
+    // early out for channels that are definitely not colourSets
+    if (name == prefToken || name == displayOpacityToken)
+      continue;
+    VtValue vtValue;
+    MColorArray colours;
+    MString colourSetName(name.GetText());
+    fnMesh.setDisplayColors(true);
+
+    if (primvar.Get(&vtValue, m_timeCode))
+    {
+
+      //early out for primvar channels that are not Vec3/Vec4 (so exclude UVs for example)
+      if (!(vtValue.IsHolding<VtArray<GfVec3f> >() || vtValue.IsHolding<VtArray<GfVec4f> >()))
+        continue;
+
+      MStatus s;
+  #if MAYA_API_VERSION >= 201800
+      colourSetName = fnMesh.createColorSetWithName(colourSetName, nullptr, nullptr, &s);
+  #else
+      colourSetName = fnMesh.createColorSetWithName(colourSetName, nullptr, &s);
+  #endif
+      if (s)
+      {
+        s = fnMesh.setCurrentColorSetName(colourSetName);
+        if (s)
+        {
+          // Prepare maya colours array
+          MFnMesh::MColorRepresentation representation;
+          if (vtValue.IsHolding<VtArray<GfVec3f> >())
+          {
+            //If we can find the special displayColorToken used by USD, let's check for the optional matching displayOpacityToken too
+            bool setCombinedDisplayAndOpacityColourSet = false;
+            if (name == displayColorToken)
+            {
+              UsdPrim prim = mesh.GetPrim();
+              if (prim.HasAttribute(primvarDisplayOpacityToken))
+              {
+                UsdAttribute usdAttr = prim.GetAttribute(primvarDisplayOpacityToken);
+                UsdGeomPrimvar primvar = UsdGeomPrimvar(usdAttr);
+                VtValue opacityValues;
+                if (primvar.Get(&opacityValues, m_timeCode))
+                {
+                  if (opacityValues.IsHolding<VtArray<float>>())
+                  {
+                    const VtArray<float> rawValOpacity = opacityValues.UncheckedGet<VtArray<float>>();
+                    colours.setLength(rawValOpacity.size());
+                    const VtArray<GfVec3f> rawValColour = vtValue.UncheckedGet<VtArray<GfVec3f> >();
+                    assert(rawValOpacity.size() == rawValColour.size());
+        
+                    for (uint32_t i = 0, n = rawValColour.size(); i < n; ++i)
+                      colours[i] = MColor(rawValColour[i][0], rawValColour[i][1], rawValColour[i][2], rawValOpacity[i]);
+                    representation = MFnMesh::kRGBA;
+                    setCombinedDisplayAndOpacityColourSet = true;
+                  }
+                }
+              }
+            }
+            if (!setCombinedDisplayAndOpacityColourSet) 
+            {
+              const VtArray<GfVec3f> rawVal = vtValue.UncheckedGet<VtArray<GfVec3f> >();
```

Comment:
why is there space between `<VtArray<GfVec3f>` and `>();`?  I've noticed this pattern has been repeated in other places in the same file. Please remove the space for all of them.